docs/consensus/genesis_config.md

```diff
@@ -30,14 +30,21 @@ let wasm_binary = WASM_BINARY.ok_or_else(|| "Wasm binary must be built for Gemin
 Specifies the type of network, highlighting its use in a development or mainnet environment.
 
 
-<!-- ## Subspace Genesis Config -->
+## Subspace Genesis Config
 
-<!-- ### sudo_account
+### sudo_account and council_members
 Specifies the account with administrative privileges, enabling essential governance functionalities.
 
 ```rust
-let sudo_account = get_account_id_from_seed("Alice");
-``` -->
+let sudo_account = AccountId::from_ss58check("5EHHtxGtDEPFX2x2PCVg8uhhg6kDdt9znQLr2oqUA9sYL5n6").expect("Wrong root account address");
```

Comment:
The current sudo is subKQqsYRyVkugvKQqLXEuhsefa9728PBAqtwxpeM5N4VD6mv (5CK2tVhvWUBnbRGsPBgQ31QDEgcEFgT7KU7FT22YLcp5A1jC).